pkg/bundle/verification_content.go

```diff
@@ -35,6 +35,10 @@ func (pk PublicKey) Hint() string {
 	return pk.hint
 }
 
+type CertificateChain struct {
+	Certificates []*x509.Certificate
+}
```

Comment:
I know there's a difference in `protobundle` between `VerificationMaterial_X509CertificateChain` and `VerificationMaterial_Certificate`, but if we had both use the `CertificateChain` type here, and had `.GetCertificateChain()` (or `.GetIntermediateCerts()` as I propose we call it) return an empty list if there's just a leaf certificate, I don't think we'd need `CertificateChain` and `Certificate` types?

@haydentherapper do you really want two separate types here, or have you [changed your mind](https://github.com/sigstore/sigstore-go/pull/101/files#r1543211390)?

Reply:
I think that'd be fine to merge the types here.

Reply:
Merged Certificate to CertificateChain